packages/web-components/src/components/cbp-usa-banner/cbp-usa-banner.scss

```diff
@@ -28,13 +29,37 @@ cbp-usa-banner {
   }
 
   .cbp-usa-banner-expand:not(#fakeId)  {
-    margin-left: var(--cbp-space-1x);
+    --cbp-button-height: 1rem;
+    --cbp-button-min-height: 1rem;
+
+    --cbp-button-padding: var(--cbp-space-2x);
+
+    --cbp-button-color: var(--cbp-color-interactive-secondary-light);
+    --cbp-button-color-hover: var(--cbp-color-interactive-secondary-lighter);
+    --cbp-button-color-focus: var(--cbp-color-violet-30);
+
+    --cbp-button-color-border-hover: transparent;
+    --cbp-button-color-border-focus: transparent;
+    --cbp-button-color-border-active: transparent;
+
+    --cbp-button-border-radius: 0;
+
+    --cbp-button-color-outline-focus: var(--cbp-color-violet-30);
 
-    --cbp-link-color: var(--cbp-color-interactive-secondary-light);
-    --cbp-link-color-hover: var(--cbp-color-interactive-secondary-lighter);
-    --cbp-link-color-focus: var(--cbp-color-violet-30);
-    --cbp-link-color-outline-focus: var(--cbp-color-violet-30);
-    --cbp-link-color-visited: var(--cbp-color-violet-1S0);
+    --cbp-button-color-bg: transparent;
+    --cbp-button-color-bg-hover: transparent;
+    --cbp-button-color-bg-focus: transparent;
+    --cbp-button-color-bg-active: transparent;
+
+    --cbp-letter-spacing-loose: none;
```

Comment:
this looks weird since you are overriding a high-level token within a selector. I would advise overriding the actual CSS property on the button in this case.